Okta.AspNet.Abstractions/UserAgentHandler.cs

```diff
@@ -4,20 +4,35 @@
 // </copyright>
 
 using System;
+using System.Net;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Okta.AspNet.Abstractions
 {
+    // TODO: consider renaming this class to OktaHttpMessageHandler, to clarify to future contributors the (expanding) responsibility of this class
     public class UserAgentHandler : DelegatingHandler
```

Comment:
I agree we should rename this class since its responsibility has changed. It could be `DefaultBackchannelHttpHandler`.

Reply:
I think calling it `DefaultBackchannelHttpHandler` would potentially be misleading.  If someone saw that name, it wouldn't be immediately obvious that this is a class provided by Okta until they looked at the namespace.  The BackChannelHandler is essentially our (Okta's) point of entry for Okta specific customization of the request pipeline.  It is conceivable that we'd need to add behavior and features in the future and this would be a likely place to put those additions.

Reply:
I agree we should consider the Okta prefix for classes/interfaces that will be exposed, but since this is an internal class I think we might want to avoid it.

Reply:
Isn't this entire repository public/exposed?  In the interest of inviting external engagement, I disagree; however, I will submit to the will of the collective.  We may want to open this up to a wider forum for input.